modules/ui/rapid_feature_toggle_dialog.js

```diff
@@ -64,7 +86,7 @@ export function uiRapidFeatureToggle(context) {
             'all-toggle',
             'rapid_feature_toggle.toggle_all', 
             null,
-            toggleAll, 
+            handleToggleAllClick, 
             drawAiFeatures.showAll(),
             false,
```

Comment:
Need to remove this comma. It is harmless from javascript purist standards, but it is causing our 'build dist' step to fail when everything is run in its entirety.